container/secrets.go

```diff
@@ -81,5 +86,8 @@ func readFile(root, name string) ([]SecretData, error) {
 }
 
 func getHostSecretData() ([]SecretData, error) {
-	return readAll("/usr/share/rhel/secrets", "")
+	if _, err := os.Stat(overrideDir); err == nil {
```

Comment:
Well if we went with the systemd way and had

/usr/share/rhel/secretes/rhsm/
filea, fileb, filec, filed
/etc/container/rhsm
filec,filef

We would end up with
filea, fileb, filed for /usr/share and filec, filef from /etc/container/